pkg/driver/options.go

```diff
@@ -32,6 +32,9 @@ type Options struct {
 	// If empty, the in-cluster config will be loaded.
 	Kubeconfig string
 
+	//RoleArn is the role driver will use to interact with the AWS EC2 APIs.
+	RoleArn string
```

Comment:
if we are adding this i think we should also update the documentation, specifically https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/docs/options.md

Reply:
Also i don't see this being set anywhere - don't we have to set this somewhere in https://github.com/sarthakkothari/aws-ebs-csi-driver/blob/e296e801d93b86ed811f28069813e1cb8c88224d/pkg/driver/options.go#L95